pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -292,15 +292,28 @@ protected TableConfig createTableConfig(File tableConfigFile)
    * Creates a new OFFLINE table config.
    */
```

Comment:
Sorry, but I had to change it because it was unreadable :) 
I'd be good to discuss current formatting rules because they seem excessive in some cases .

Reply:
I'm with you on that. These fluent method calls should (almost) always be written on separated lines, although there is no consensus right now and it is specially important to have a config file that configures IDEs to respect the indentation.

Meanwhile please add a `//@formatter:off` and `//@formatter:on` and and before the code block to skip future modifications.